daemon/create.go

```diff
@@ -96,7 +96,7 @@ func (daemon *Daemon) create(params types.ContainerCreateConfig, managed bool) (
 		return nil, err
 	}
 
-	if container, err = daemon.newContainer(params.Name, params.Config, imgID, managed); err != nil {
+	if container, err = daemon.newContainer(params.Name, params.Config, params.HostConfig.NetworkMode.IsHost(), imgID, managed); err != nil {
```

Comment:
Should `params.HostConfig` be passed here, instead of a Boolean? Maybe @estesp has some thoughts on that as well, so don't change yet (just thinking out loud)